gce/prepull-1.21.yaml

```diff
@@ -36,6 +36,42 @@ spec:
       # run ping in each container to keep it alive so that kubernetes does not
       # continually restart the containers while we're prepulling.
       containers:
+      - image: k8s.gcr.io/e2e-test-images/agnhost:2.26
```

Comment:
Shouldn't we cleanup the other images / entries from this file? We won't need them in tests, and it will only add unnecessary execution time to the prepull.

Reply:
My concern here is that LTSC 2019 and 1909 run off the old image lists. So it's unclear how to proceed there.

Reply:
All published images in the ``k8s.gcr.io/e2e-test-images`` registry contains the following OS versions: 1809, 1903, 1909, 2004, 20H2. You can even check this here: https://github.com/kubernetes/kubernetes/blob/master/test/images/agnhost/BASEIMAGE

This might change some time in the future, depending on what the community decides to test and support in the future.

Reply:
The repo lists for older LTSC 2019 and 1909 is pointing to the non 2004 repo list. Should they also be updated to the -2004 repo list?

Reply:
Are the base layers underneath the new `k8s.gcr.io/e2e-test-images` images and the existing `k8sprow.azurecr.io/kubernetes-e2e-test-images` images identical? If so then it won't matter anyway.